src/native/src/3rdparty/json/ChangeLog.md

```diff
@@ -0,0 +1,224 @@
+# Change Log
```

Comment:
I think longer clone times are better than littered source. As an alternative it can in the make file or some preparation script to clone the dependency with --depth=1 and .gitignore that directory so it's not accidentally added.